sidebars.js

```diff
@@ -39,6 +38,7 @@ const sidebars = {
       type: "category",
       label: "Core Concepts",
       items: [
+        "core-concepts/authenticating-with-api-keys",
```

Comment:
Does this mean that the API key is the first thing in the core concepts? I'm not too big of a fan of API keys showing up as a "Core Concept" at all though, to me core concepts are things about how we are solving their problems in automated QA, and I don't really consider the ability to authenticate as a core feature they would care about in that context. Maybe this should be in its own Account Management section?

Reply:
Why don't we see how the other pages flesh out and then we can move things around more if this doesn't make sense to keep here? Updating links to pages and images has been time-consuming and I'm more concerned with getting the content _out there_ for people to use